apps/web/src/state/farmsV3/hooks.ts

```diff
@@ -66,7 +67,7 @@ export const useFarmsV3Public = () => {
     [chainId, 'farmV3ApiFetch'],
     async () => {
       if (API_FLAG) {
-        return farmV3ApiFetch(chainId).catch((err) => {
+        return farmV3ApiFetch(chainId ?? -1).catch((err) => {
```

Comment:
Should throw error if chainId is not defined?

Reply:
I update it to this 
      if (API_FLAG && chainId) {
        return farmV3ApiFetch(chainId).catch((err) => {
          console.error(err)
          return fallback
        })
      }